core/src/ch/asynk/gdx/boardgame/Tile.java

```diff
@@ -41,4 +41,19 @@ public void enableOverlay(int i, boolean enable)
     {
         return "[" + x + ", " + y + "]";
     }
+
+    @Override public boolean equals(Object o)
+    {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Tile tile = (Tile) o;
+        return Float.compare(tile.x, x) == 0 &&
+                Float.compare(tile.y, y) == 0 &&
+                (overlays == tile.overlays) || (overlays != null && overlays.equals(tile.overlays));
```

Comment:
overlays shouldn't be used here, a tile is defined by it's coordinates only.

Reply:
Nice idea, but how would you do that?